src/applications/vre/definitions/profileAddress.js

```diff
@@ -0,0 +1,265 @@
+/**
+ * This uiSchema is modeled after how addresses are handled in the Profile application, and makes the same pattern
+ * available for use inside forms generated by the platform's form system.
+ */
+
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+import ADDRESS_DATA from 'platform/forms/address/data';
+import get from 'platform/utilities/data/get';
+import {
+  countries,
+  states50AndDC,
+  militaryCities,
+} from 'vets-json-schema/dist/constants.json';
+
+/**
+ * CONSTANTS:
+ * 1. MILITARY_STATES - contains military state codes and names.
+ * 2. USA - references USA value and label
+ * 3. MilitaryBaseInfo - React component. Wrapped in AdditionalInfo component and used as description
+ */
+
+// Corresponding working form schema:
+export const addressSchema = {
+  type: 'object',
+  properties: {
+    isMilitary: {
+      type: 'boolean',
+    },
+    'view:livesOnMilitaryBaseInfo': {
+      type: 'object',
+      properties: {},
+    },
+    country: {
+      type: 'string',
+      enum: countries.map(country => country.value),
+      enumNames: countries.map(country => country.label),
+    },
+    addressLine1: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 100,
+      pattern: '^.*\\S.*',
+    },
+    addressLine2: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 100,
+      pattern: '^.*\\S.*',
+    },
+    addressLine3: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 100,
+      pattern: '^.*\\S.*',
+    },
+    city: {
+      type: 'string',
+    },
+    state: {
+      type: 'string',
+    },
+    postalCode: {
+      type: 'string',
+    },
+  },
+};
+const MILITARY_STATES = Object.entries(ADDRESS_DATA.states).reduce(
```

Comment:
This is already defined: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/address.js#L252-L256

Reply:
excellent